deep_quoridor/src/agents/simple.py

```diff
@@ -135,7 +125,19 @@ def choose_action(
             game.board.move_player(current_player_before_action, position_before_action)
         game.set_current_player(current_player_before_action)
 
-    return chosen_action, chosen_value
+    # "player" tries to maximize the reward, while "opponent" tries to minimize it.
+    values = np.array(values)
+    if current_player_before_action == player:
+        best_value = values.max()
+    else:
+        best_value = values.min()
+
+    best_action_indices = np.flatnonzero(values == best_value)
+    # There may be multiple actions with the same value, so we randomly choose one of them.
+    # chosen_action_index = np.random.choice(best_action_indices)
```

Comment:
Any reason to use the last rather than picking a random one?

Reply:
Ah, great catch! Didn't mean to leave that in; fixed now. Interestingly fixing that makes simple a bit better as P2 but a bit worse as P1.....